File: Challenge#01/Challenge_01.py
text = []
with open("Challenge#01/message_01.txt", "r") as file:
    f = list(file)
    for word in f:
        word = word.lower()
        text += word.split(" ")

result = []
response = ""
for word in text:
    if word not in result:
        result.append(word)
        response += word
        response += str(text.count(word))
print(response)

File: Challenge#01/Challenge_01.js
const fs = require('fs');
const { workerData } = require('worker_threads');

const input = fs.readFileSync('Challenge#01/message_01.txt', 'utf8').split('\n');
const words_split = input[0].split(' ');

let words = [];

for (let elements of words_split) {
    if (!words.includes(elements)) {
        words.push(elements);
        words.push(words_split.filter(x => x === elements).length)
    }
}

words = words.join('');
console.log(words)
